eos-parental-controls-client/eos-parental-controls-client.py

```diff
@@ -95,6 +95,23 @@ def __lookup_user_id_or_error(user):
         raise SystemExit(EXIT_INVALID_OPTION)
 
 
+def __oars_value_to_string(value):
+    """Convert an EosParentalControls.AppFilterOarsValue to a human-readable
+    string."""
+    mapping = {
+        EosParentalControls.AppFilterOarsValue.UNKNOWN: "unknown",
+        EosParentalControls.AppFilterOarsValue.NONE: "none",
+        EosParentalControls.AppFilterOarsValue.MILD: "mild",
+        EosParentalControls.AppFilterOarsValue.MODERATE: "moderate",
+        EosParentalControls.AppFilterOarsValue.INTENSE: "intense",
+    }
+
+    try:
+        return mapping[value]
```

Comment:
`return value.value_nick`?

Reply:
Hmm, I’d prefer not to because the nick is not (potentially) translatable in future. Its semantics/purpose are undefined, and hence may not be well suited to this particular instance of string printing.